src/guacscanner/guacscanner.py

```diff
@@ -708,8 +709,8 @@ def main() -> None:
         }
     )
     try:
-        validated_args = schema.validate(args)
-    except SchemaError as err:
+        validated_args = mySchema.validate(args)
```

Comment:
Paired with the above suggestion.

```suggestion
        validated_args = my_schema.validate(args)
```